faasmcli/faasmcli/tasks/upload.py

```diff
@@ -1,36 +1,22 @@
-import multiprocessing
-from functools import partial
-from os import makedirs, listdir
+from os import makedirs
```

Comment:
Faasm no longer has the concept of functions held in this repo, so the upload task is now much simpler (just points at a file)